ccmlib/scylla_repository.py

```diff
@@ -178,16 +178,17 @@ def download_version(version, url=None, verbose=False, target_dir=None):
         package_version_file = "{}/.relocatable_package_version".format(target_dir)
         if os.path.exists(package_version_file):
             with open(package_version_file) as f:
-                package_version = f.read().strip()
-            if package_version != '2':
-                print('Unknown relocatable package format version: ' + package_version)
+                package_version = int(f.read().strip())
```

Comment:
should we consider using something like LooseVersion instead of int ? so later we could have things like `1.2` ?